### ISSUE TITLE
Investigate and fix daily a11y scan permission errors

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/va.gov-team#45928]
- [Daily a11y scan is failing.](https://dsva.slack.com/archives/C01RAS1KAQK/p1660749737623059)
- Checkout content-build step has permission errors: `warning: failed to remove (filename): Permission denied`

**Comment by rsmithadhoc** [comment]
- I'm not having much luck so far. I've tried solutions found online to try and clean up those files but none have worked so far. I have a few more things to try.

**Comment by rsmithadhoc** [comment]
- It is a more widespread issue with the runner, release tools also had this issue.
- Sri is looking into it. If he does not come up with anything, we will have to reach out to infra.
- Nothing more we can do at the moment.

**Comment by rsmithadhoc** [comment]
Darius made an update and the daily scan is running successfully now, closing this out.
